lib/phoenix/live_dashboard/page_builder.ex

```diff
@@ -106,7 +106,7 @@ defmodule Phoenix.LiveDashboard.PageBuilder do
   A page can only have the components listed with this page.
 
   We currently support `card/1`, `columns/1`, `fields_card/1`,
-  `layered_graph/1`, `nav_bar/1`, `row/1`, `shared_usage_card/1`, `table/1`,
+  `live_layered_graph/1`, `live_nav_bar/1`, `row/1`, `shared_usage_card/1`, `live_table/1`,
```

Comment:
Probably we want to rewrite the documentation, not sure if we want to do it in this PR or in a new one.

Reply:
As long as we do immediately after, it is fine.